README.md

```diff
@@ -34,16 +34,16 @@ Once built the OCI Image can then be discovered by name, downloaded, verified by
 
 ## Understanding the Specification
 
-The [Media Types](media-types.md) document is a starting point to understanding the overall structure of the specification. This document outlines the OCI Image file format specifications, including the [filesystem layer changesets](layer.md) and image manifest described above.
+The [OCI Image Media Types](media-types.md) document is a starting point to understanding the overall structure of the specification.
 
-The high level components of the spec include:
+The high-level components of the spec include:
 
-* An [image manifest](manifest.md), a set of [filesystem layers](layer.md), and [image configuration](config.md) (base layer)
-* A process of hashing the image format for integrity and content-addressing (base layer)
-* Signatures that are based on signing image content address (optional layer)
-* Naming that is federated based on DNS and can be delegated (optional layer)
+* An archival format for container images, consisting of an [image manifest](manifest.md), an [image layout](image-layout.md), a set of [filesystem layers](layer.md), and [image configuration](config.md) (base OCI layer)
```

Comment:
Eh, this sentence is already unwieldy - I'd rather let manifest.md be the canonical one and there are links to the manifest list from there